src/OptimisticTokenVotingPlugin.sol

```diff
@@ -539,6 +586,26 @@ contract OptimisticTokenVotingPlugin is
         });
     }
 
+    /// @notice Internal function to update the excluded voting power holders.
+    /// @param _excludedVotingPowerHolders The list of addresses to be excluded from voting power.
+    function _updateExcludedVotingPowerHolders(address[] calldata _excludedVotingPowerHolders) internal {
```

Comment:
As it is, the function only allows to append addresses. If we call it "update", then we should allow to add/remove or to simply "set" the list of addresses